tests/ro-crate-metadata.json

```diff
@@ -0,0 +1,457 @@
30b3af01f3e58c8c79413\",\n        \"size\": 592394,\n        \"path\": \"/Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small_fastqc.html\"\n    },\n    \"qc_result_2\": {\n        \"location\": \"file:///Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_2.small_fastqc.html\",\n        \"basename\": \"ERR034597_2.small_fastqc.html\",\n        \"class\": \"File\",\n        \"checksum\": \"sha1$db73f100c14f0c9a1dad40af5a0755c60d434912\",\n        \"size\": 592566,\n        \"path\": \"/Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_2.small_fastqc.html\"\n    },\n    \"trimmed_fastq1P\": {\n        \"location\": \"file:///Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small.fq.trimmed.1P.fq\",\n        \"basename\": \"ERR034597_1.small.fq.trimmed.1P.fq\",\n        \"class\": \"File\",\n        \"checksum\": \"sha1$5188cdb59cab8010eb673e8bc113b7fd7a686660\",\n        \"size\": 5566650,\n        \"path\": \"/Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small.fq.trimmed.1P.fq\"\n    },\n    \"trimmed_fastq1U\": {\n        \"location\": \"file:///Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small.fq.trimmed.1U.fq\",\n        \"basename\": \"ERR034597_1.small.fq.trimmed.1U.fq\",\n        \"class\": \"File\",\n        \"checksum\": \"sha1$be4d3444537d2596d122db1ca8e5955954094f6b\",\n        \"size\": 187289,\n        \"path\": \"/Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small.fq.trimmed.1U.fq\"\n    },\n    \"trimmed_fastq2P\": {\n        \"location\": \"file:///Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small.fq.trimmed.2P.fq\",\n        \"basename\": \"ERR034597_1.small.fq.trimmed.2P.fq\",\n        \"class\": \"File\",\n        \"checksum\": \"sha1$cd1451093c91a9b619337a7bfbe592ad76554745\",\n        \"size\": 5560582,\n        \"path\": \"/Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small.fq.trimmed.2P.fq\"\n    },\n    \"trimmed_fastq2U\": {\n        \"location\": \"file:///Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small.fq.trimmed.2U.fq\",\n        \"basename\": \"ERR034597_1.small.fq.trimmed.2U.fq\",\n        \"class\": \"File\",\n        \"checksum\": \"sha1$dd7edf1b2d599eb3c8fa49426be55d8a0360561c\",\n        \"size\": 80131,\n        \"path\": \"/Users/inutano/repos/sapporo-wes/sapporo-service/run/26/26cee039-7a37-4647-924f-ee2187304ef3/outputs/ERR034597_1.small.fq.trimmed.2U.fq\"\n    }\n}\n"
```

Comment:
This is not necessary if the file is included in the crate

Reply:
This is what we use in our application Tonkaz to compare the outputs of two workflow runs. Since we just want to compare the ro-crate files, not by transferring and comparing entire output files, we need to record these kinds of stats in a crate